lib/render/ContourRenderer.cpp

```diff
@@ -116,7 +116,11 @@ int ContourRenderer::_buildCache()
     double mv = grid->GetMissingValue();
     float  Z0 = GetDefaultZ(_dataMgr, _cacheParams.ts);
 
-    Grid::ConstCellIterator it = grid->ConstCellBegin(_cacheParams.boxMin, _cacheParams.boxMax);
+    CoordType boxMin = {0.0, 0.0, 0.0};
+    CoordType boxMax = {0.0, 0.0, 0.0};
+    Grid::CopyToArr3(_cacheParams.boxMin, boxMin);
+    Grid::CopyToArr3(_cacheParams.boxMax, boxMax);
+    Grid::ConstCellIterator it = grid->ConstCellBegin(boxMin, boxMax);
```

Comment:
Instead of having to repeat this process every time we want to get a grid, couldn't `ConstCellBegin` be overloaded to support vectors? Especially since `Box` still returns vectors.

Reply:
Indeed. Done.